### PR TITLE
feat: update react native in example to 0.69.2

### PR DESCRIPTION
**Pull request by alanjcharles**
Updates react native version in example app to `0.69.2` to support new react native architecture

### CODE REVIEW

**Comment by oscb** [comment]
Looks good. Only request can you also update the readme here for the instructions on how to import correctly and setup the trackdeeplinks after this change?

**Comment by alanjcharles** [comment]
@oscb thanks! Definitely can do.

File: example/package.json [hunk 1]
```diff
@@ -24,9 +24,9 @@
     "@react-native-community/masked-view": "^0.1.11",
     "@react-navigation/native": "^6.0.2",
     "@react-navigation/stack": "^6.0.7",
-    "@segment/sovran-react-native": "^0.4.0",
-    "react": "17.0.2",
-    "react-native": "0.67.3",
+    "@segment/sovran-react-native": "0.4.2",
```
Comment (oscb) [inline example/package.json:27]: Upgrade also the package.json in core to use this specific version of sovran

**Review by oscb — CHANGES_REQUESTED** [review]
Just one change needed in core

**Review by oscb — APPROVED** [review]

**Review by oscb — APPROVED** [review]

**Pull request merged**